### ISSUE TITLE
NOTICE, FATAL messages in stderr

### ISSUE THREAD

**Issue by gajus** [brianc/node-postgres#2006]
My stderr output is flooded with lines starting `NOTICE:`, `FATAL:`, etc. e.g.

```
NOTICE:  next maintenance_task ID 364
NOTICE:  scheduling execution of 364 maintenance_task
NOTICE:  next maintenance_task ID 68
NOTICE:  execution duration 00:00:00.132437s
NOTICE:  scheduling execution of 68 maintenance_task
NOTICE:  next maintenance_task ID 331
NOTICE:  scheduling execution of 331 maintenance_task
NOTICE:  next maintenance_task ID 532
FATAL:  terminating connection due to idle-in-transaction timeout

```

Where is this coming (I don't see it in `node-postgres` codebase, I am assuming it is one of the dependencies) and how to disable it?

**Comment by boromisp** [comment]
These look like typical postgres log entries. What you need to look for is `notice` event handlers for clients and maybe the pool. If nothing changed in your code, maybe the database configuration or version has changed?

**Comment by brianc** [comment]
are you using the native driver? did you ever find out what was going on here?

**Comment by gajus** [comment]
> are you using the native driver? did you ever find out what was going on here?

Yes, I am. I tried grepping node_modules but couldn't find where it originates. I have just ignored it though as it wasn't having any impact.
